### PR TITLE
Fixes #1101 Set API_HOST env and cp .env.example to .env

### LINKED ISSUE #1101: Set API_HOST env and cp .env.example to .env
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**

Currently, API_HOST isn't set and doc to copy env variables

**Expected behavior:**

Doc to export API_HOST env variable and copy envs.

**Steps to reproduce:**

**Related code:**

```
insert any relevant code here else remove this section
```

**Other information:**

**System information:** 

```
Your operating system
```

```
output of `node -v`
```

```
output of `npm ls ember-cli`
```

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by bhaveshAn**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Setting environment variables `API_HOST` and that in `.env.example`

#### Changes proposed in this pull request:

- Setting environment variables `API_HOST` and that in `.env.example`
-
-

Fixes #1101 Set API_HOST env and cp .env.example to .env

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ It is also recommended to have [watchman](https://facebook.github.io/watchman/do
 * `git clone <repository-url>` this repository
 * `cd open-event-frontend`
 * `yarn`
+Set Environment variable API_HOST which is running as backend at http://localhost:5000 (Open Event Server)
+* `export API_HOST=http://localhost:5000`
```
Comment (niranjan94) [inline README.md]: Instead of this, you can add it to `.env.example` and let the user decide what they want to do.

Comment (bhaveshAn) [reply README.md]: Please check now.

**Comment by CosmicCoder96** [comment]
@bhaveshAn Why 3 commits with identical messages ? 🤔

**Comment by bhaveshAn** [comment]
i just forgot to squash it at that time. Will take car from now.

**Pull request merged**